packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -305,14 +313,11 @@ export default {
 .spaces-list {
   &-card {
     box-shadow: none !important;
+    border-radius: 3px;
+    background-color: var(--oc-color-background-highlight) !important;
 
-    .space-context-btn {
-      z-index: 999;
-    }
-
-    .oc-card-media-top button {
-      top: 0;
-      right: 0;
+    .oc-card-body {
+      padding: 10px;
```

Comment:
possible to use the `m` sized spacing css variable? It's 8px. But IMO it still looks good.

Reply:
Basically: avoid using `px` values anywhere you can.